client-cli/src/command/wallet_command.rs

```diff
@@ -35,6 +35,13 @@ pub enum WalletCommand {
             case_insensitive = true
         )]
         wallet_type: WalletKind,
+        #[structopt(
+            name = "wallet mnemonic count",
+            short = "m",
+            long = "mnemonics_word_count",
+            help = "Number of words in mnemonics"
+        )]
+        mnemonics_word_count: u32,
```

Comment:
Can user enter any value here (other than `12` and `24`)? If not, Maybe it's better to have an enum `MnemonicType` with variants `Long` and `Short` to represent this? Also, set the default value to `Long`.

Reply:
also support
12, 15,18,21,24
because it's word count, just using value seems easier.

but using enum also make sense.
then enum value will be represented as `String`